.github/workflows/snapshot.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     permissions:
       id-token: write
     runs-on: ubuntu-latest
+    timeout-minutes: 20
```

Comment:
Ooh, worth adding to our template?

Reply:
Ooo probably, I accidentally hung the actions for a hour or 2 😅